modules/apps/frontend-theme-locos/locos-theme/src/css/_content.scss

```diff
@@ -0,0 +1,72 @@
+
+@import "_navigation";
+@import "_breadcrumb";
+@import "_animation";
+@import "_gallery";
```

Comment:
Alphabetize CSS imports.  If it needs to be in a specific order, add a line break between each file.